src/components/ShareApplet/SuccessShared/SuccessShared.tsx

```diff
@@ -46,19 +47,17 @@ export const SuccessShared = ({
         <StyledAppletContent>
           <StyledLabelLarge>{title}</StyledLabelLarge>
           <StyledText color={variables.palette.on_surface_variant}>{text}</StyledText>
-          {keywords.length > 0 && (
-            <StyledChipsWrapper sx={{ marginTop: theme.spacing(0.8) }}>
+          {keywords?.length > 0 && (
+            <StyledFlexWrap sx={{ marginTop: theme.spacing(0.8) }}>
               {keywords.map((word, i) => (
                 <Chip color="secondary" key={i} title={word} />
               ))}
-            </StyledChipsWrapper>
+            </StyledFlexWrap>
           )}
-          {activitiesQuantity ? (
+          {activitiesQuantity && (
```

Comment:
in case of activitiesQuantity=0 here will be 0

Reply:
> in case of activitiesQuantity=0 here will be 0

fixed